pytest_html/resources/main.js

```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+//created to increase testability
+var  DOCUMENT = document;
+
+function setDocument(elem) {
+  DOCUMENT = elem;
+}
```

Comment:
@davehunt I would like to know if you think that is a good practice, because I could not mock document by another way, and I thought that it maybe would be a not that good practice.